testing/cdms2/CMakeLists.txt

```diff
@@ -92,6 +92,10 @@ cdat_add_test("CDMS_Test_18"
     "${PYTHON_EXECUTABLE}"
     ${cdat_SOURCE_DIR}/testing/cdms2/cdtest18.py)
 
+cdat_add_test("CDMS_Test_20"
```

Comment:
All the other one are called CDMS_Test_XX!  Not sure what happened to CDMS_Test_19 but it was disabled and cdtest19.py exist in testing/cdms2.
```
 cdat_add_test("CDMS_Test_16"
     "${PYTHON_EXECUTABLE}"
     ${cdat_SOURCE_DIR}/testing/cdms2/cdtest16.py)
 
 cdat_add_test("CDMS_Test_17"
     "${PYTHON_EXECUTABLE}"
     ${cdat_SOURCE_DIR}/testing/cdms2/cdtest17.py)
 
 cdat_add_test("CDMS_Test_18"
     "${PYTHON_EXECUTABLE}"
     ${cdat_SOURCE_DIR}/testing/cdms2/cdtest18.py)
```